yt_idefix/fields.py

```diff
@@ -47,3 +62,266 @@ def setup_fluid_fields(self):
+        unit_system = self.ds.unit_system
+
+        setup_magnetic_field_aliases(
+            self, self.ds._dataset_type, [f"Bx{idir}" for idir in "123"]
+        )
+        for idir in "123":
+            self.alias(
+                ("gas", f"BX{idir}"),
+                (self.ds._dataset_type, f"Bx{idir}"),
+                units="code_magnetic",
+            )
+
+        # Add tracer fields
+        for i in range(1, self.ds.ntracers + 1):
+            if (self.ds._dataset_type, "tr%d" % i) in self.field_list:
+                self.add_output_field(
+                    (self.ds._dataset_type, "tr%d" % i),
+                    sampling_type="cell",
+                    units="",
+                )
+                self.alias(
+                    ("gas", "Tracer %d" % i),
+                    (self.ds._dataset_type, "tr%d" % i),
+                    units="",
+                )
+                self.alias(
+                    ("gas", "tracer %d" % i),
+                    (self.ds._dataset_type, "tr%d" % i),
+                    units="",
+                )
+
+        if (self.ds._dataset_type, "Temp") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "Temp"),
+                sampling_type="cell",
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "Temperature"),
+                (self.ds._dataset_type, "Temp"),
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                (self.ds._dataset_type, "Temp"),
+                units=unit_system["temperature"],
+            )
+        elif (self.ds._dataset_type, "temp") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "temp"),
+                sampling_type="cell",
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "Temperature"),
+                (self.ds._dataset_type, "temp"),
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                (self.ds._dataset_type, "temp"),
+                units=unit_system["temperature"],
+            )
+        elif (self.ds._dataset_type, "Temperature") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "Temperature"),
+                sampling_type="cell",
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "Temperature"),
+                (self.ds._dataset_type, "Temperature"),
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                (self.ds._dataset_type, "Temperature"),
+                units=unit_system["temperature"],
+            )
```

Comment:
```suggestion
```

Reply:
Temp and Temperature are most commonly used in the PLUTO community. I have changed this accordingly.